### PR TITLE
refactor php phases to be separate layer_builders

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

File: ftl/php/builder.py [hunk 1]
```diff
@@ -45,21 +45,33 @@ def Build(self):
         lyr_imgs.append(self._base_image)
         if ftl_util.has_pkg_descriptor(self._descriptor_files, self._ctx):
             pkgs = self._parse_composer_pkgs()
-            # if there are 42 or more packages, revert to using phase 1
+            # due to image layers limites, we revert to using phase 1 if over
```
Comment (nkubala) [inline ftl/php/builder.py]: nit: s/limites/limits

File: ftl/php/builder.py [hunk 2]
```diff
@@ -45,21 +45,33 @@ def Build(self):
...
                     descriptor_files=self._descriptor_files,
-                    pkg_descriptor=pkg_txt,
                     destination_path=self._args.destination_path,
                     cache=cache)
                 layer_builder.BuildLayer()
                 lyr_imgs.append(layer_builder.GetImage())
+            else:
+                # phase 2
+                for pkg_txt in pkgs:
+                    logging.info('Building package layer')
```
Comment (nkubala) [inline ftl/php/builder.py]: nit: combine these two lines

```logging.info('Building package layer: {0}'.format(pkg_txt))```

File: ftl/php/builder.py (hunk 1 above)
Comment (aaron-prindle) [reply ftl/php/builder.py]: done

File: ftl/php/builder.py (hunk 2 above)
Comment (aaron-prindle) [reply ftl/php/builder.py]: done

**Pull request merged**